packages/server/src/errorNormalize.ts

```diff
@@ -0,0 +1,95 @@
+// The functions in this file are not part of Apollo Server's external API.
```

Comment:
Seems like this still belongs in `src/errors/` but fine either way

Reply:
Hmm. My thought was that since this code is entirely consumed outside of that sub-export, then it didn't make sense for it to be in the subdirectory?  I guess that the tree shaking bundlers we care about probably won't scoop up those files if you don't do the top export, but since the whole point is to minimize `src/errors` to the stuff that clients might need it seemed like a bad idea to put unrelated stuff there. I thought about doing src/internalErrors or something but it seemed like overkill. Thoughts?

Reply:
Fine as is, didn't feel too strongly about this. Agree with not doing an `internalErrors` folder.